src/databases/Silo/avtSiloFileFormat.C

```diff
@@ -10412,25 +10412,27 @@ avtSiloFileFormat::ReadInConnectivity(vtkUnstructuredGrid *ugrid,
                     {
                     case VTK_QUADRATIC_WEDGE:
                     {
-                        nblinnod = 6 ;
-                        vtkIdType vtk_wedge[nblinnod];
+                        nblinnod = 6;
```

Comment:
```suggestion
                       int const nblinnod = 6;
```

Deleting line 10406 and instead doing this here and other places would alleviate need for matching `new`/`delete` calls which I think would be better.